api/specs/storage/Makefile

```diff
@@ -6,13 +7,12 @@ _check_venv_active:
 
 
 .PHONY: install
-install: _check_venv_active
-	@cd ./../../../../services/storage && make install-dev && cd -
+install-dev install: _check_venv_active
+	# installing storage and requirements.txt
+	@cd ./../../../services/storage && make install-dev && cd -
 	@uv pip install -r requirements.txt
 
 
 .PHONY: all
-all: _check_venv_active
-	@for file in *.py; do \
-		python $$file; \
-	done
+all: _check_venv_active install
+	python openapi.py
```

Comment:
Thought: we should start using `uv --script` or `uv tool`

Reply:
You read my mind :-) I tried it but it is not that straighforward.
The problem here is that that script also needs `simcore_service_webserver` (and all its dpendencies). Any hints on how to do this?

Reply:
https://github.com/ITISFoundation/osparc-simcore/issues/6810